src/constants/contracts/contracts.ts

```diff
@@ -31,6 +38,13 @@ type Contract = {
   address: Address
 }
 
+/**
+ * Retrieves all contracts.
+ *
+ * @returns {Array<ContractConfig>} An array containing the contracts' ABI and addresses.
+ */
+export const getContracts = () => contracts
```

Comment:
Should we use `as const` in contracts to improve the type inference (removing the implicit annotation on L:21)?

Reply:
but you lose the type check when you update the `contracts` object, right?